Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorHelpers.cpp

```diff
@@ -63,23 +63,22 @@ namespace AzToolsFramework
     static const float s_iconFarDist = 40.f; // distance at which icons are at minimum scale
 
     // helper function to wrap EBus call to check if helpers are being displayed
-    // note: the ['?'] icon in the top right of the editor
     static bool HelpersVisible()
     {
         bool helpersVisible = false;
-        EditorRequestBus::BroadcastResult(helpersVisible, &EditorRequests::DisplayHelpersVisible);
+        EditorRequestBus::BroadcastResult(helpersVisible, &EditorRequestBus::Events::DisplayHelpersVisible);
```

Comment:
I wonder...
```cpp
EditorRequestBus::Broadcast([&](auto *iface) { helpersVisible = iface->DisplayHelpersVisible(); });
```
:smile:

Reply:
Haha nice idea, I always forget about the lambda version of `Broadcast` - I can update this when I publish the other tests :+1: